core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -2588,6 +2588,7 @@ final class Stream[+F[_], +O] private[fs2] (private[fs2] val underlying: Pull[F,
     * res0: List[Chunk[Int]] = List(Chunk(1), Chunk(2), Chunk(3), Chunk(4), Chunk(5), Chunk(6))
     * }}}
     */
+  @deprecated("Use .chunkLimit(1).unchunks instead.", "3.0.7")
```

Comment:
This should have introduced a bunch of deprecation warnings in the test suite. Can we address those too?

Reply:
On it.